reporter-web-app/src/components/PackageDetails.js

```diff
@@ -60,7 +60,7 @@ const PackageDetails = (props) => {
                 {id}
             </Item>
             {
-                purl.length !== 0
+                !!purl
```

Comment:
Can you give me a hint about the semantics of `!!`?

In particular also, is the length check still covered?

Reply:
! forces a boolean from any type and !! inverts the first evaluated boolean e.g. not(true) / not (false). The original condition purl.length !== 0 check if purl string was greater than zero but did not check if purl was undefined that why we got type error can determine length of undefined. Using !! I no check undefined and string length in one statement as in JavaScript boolean(undefined) == false, boolean('') == false.

Reply:
Thanks!